css/properties/grid-template-rows.json

```diff
@@ -322,7 +322,7 @@
                 "version_added": "10.3"
               },
               "samsunginternet_android": {
-                "version_added": null
+                "version_added": "6.0"
```

Comment:
```suggestion
                "version_added": "7.0"
```

Reply:
CSS Grid landed in 6.0 not 7.0 in Samsung Internet